### PR TITLE
Better 'developer getting started' docs

### PR DESCRIPTION
**Pull request by DanVanAtta**
Perhaps most notable addition is: docs/development/tutorial/README.md , which gives some practical steps to build and run the code (beyond just getting set up in IDE)

### CODE REVIEW

File: docs/development/how-to/ide-setup/intellij-setup.md [hunk 1]
```diff
@@ -32,3 +45,4 @@ Note: The below instructions reference IntelliJ Settings, which are accessible v
   1. *File > Import Settings*
```
Comment (emmanuel-perlow) [inline docs/development/how-to/ide-setup/intellij-setup.md:45]: On my Linux system (version ideaIC-2021.2) importing settings is found under `File > Manage IDE Settings > Import Setting`

File: docs/development/tutorial/README.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
+# NEW DEVELOPER GETTING STARTED!
+
+## Assumptions
+
+- Some familiarity with 'git' is assumed
+- Some familiarity with a favorite
```
Comment (emmanuel-perlow) [inline docs/development/tutorial/README.md:6]: Broken sentence?

File: docs/development/tutorial/README.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+- [/docs/development/how-to/typical-git-workflow](../how-to/typical-git-workflow.md)
+- [/docs/development/reference/dev-process/pull-requests.md](../reference/dev-process/pull-requests.md)
+
+
+
+## Running the code
+
+- IDE (there are launchers checked in for convenience)
+- CLI: `./gradlew :game-app:game-headed:run`
```
Comment (emmanuel-perlow) [inline docs/development/tutorial/README.md:27]: I guess this is Linux (and potentially MacOS) only.
The sources contains a `gradlew.bat` which I guess is the Windows equivalent?

**Comment by emmanuel-perlow** [comment]
LGTM. Thanks for the write up!

I would add that on the first invocation `gradlew` (or the IDE running gradle) downloads a lot of stuff (among others a JDK and alll the dependencies) and thus needs a working internet connection.

File: docs/development/tutorial/README.md (hunk 2 above)
Comment (DanVanAtta) [reply docs/development/tutorial/README.md]: Thanks for finding this :+1: 
Yeah, just bad editting here.

File: docs/development/tutorial/README.md (hunk 3 above)
Comment (DanVanAtta) [reply docs/development/tutorial/README.md:27]: `gradlew.bat` is indeed the windows equivalent
I updated assumptions to note that we presume someone is going to go ahead and run 'cygwin'. If the windows CLI command is really straight forward and works just fine, it'd be fine to add to the documents. 

Notably it's the responsibility of anyone to ensure any setup and configuration that they use is documented. This avoids others from having to again re-invent what is already known by someone else. You might tell that I use linux (and happily Mac OS is basically the same from a CLI point of view).

**Comment by DanVanAtta** [comment]
@emmanuel-perlow that would be a great comment to add in regarding first invocation of gradlew. This was just merged, would you like to make that update as practice perhaps?

**Pull request merged**